src/auto/key_file.rs

```diff
@@ -283,7 +293,12 @@ impl KeyFile {
         }
     }
 
-    //pub fn to_data(&self) -> Result<(String, /*Unimplemented*/Fundamental: Size), Error> {
-    //    unsafe { TODO: call ffi::g_key_file_to_data() }
-    //}
+    pub fn to_data(&self) -> Result<(String, usize), Error> {
```

Comment:
Yeah, just saying :) In my local branch this ends up being `pub fn to_data(&self) -> Result<String, Error>` already btw. Hope to be able to clean that up a bit for an initial review later today.